### DISCUSSION TITLE
Project specific configuration location

### DISCUSSION THREAD

**Discussion by Rintse** [haskell/haskell-language-server#1522] [Q&A] [answered]
HLS in coc (nvim) is helping me through my first big haskell project very nicely, but I accidentally suppressed a certain kind of error (I think its only for the file i was in).
I've been looking all over for some sort of command or config file to revert this, but I have been unable to find one.

How would I be able to revert/edit my file/project specific settings?

EDIT: the suppression was done through a coc-codeaction

**Comment by jneira** [comment, marked as answer]
Hi! Afaik you cant disable a specific error in a hls specific configuration you might do it adding some language extension or other pragma in the file if the error was from ghc. is the error shown if you build the project out of hls (with `cabal build` or `stack build`)?

**Reply by Rintse** [reply]
For this one specific file, hls does not show the errors, but when `stack build`-ing, they do appear and I have to fix them switching back and forth from my editor.

**Reply by jneira** [reply]
I see, thanks for confirming it, could you add a link to the project (or a minimal repro one) to take a look to the module, stack and .cabal config, etc?

**Reply by Rintse** [reply]
Sure thing.
https://github.com/Rintse/hopfl
Thanks for looking into this.

**Reply by Rintse** [reply]
Oh whoops, you're totally right, it's just a pragma.
I could have sworn I looked for those and couldn't find them, but apparently my eyes are just defective.

Leaves me to wonder how `stack build` is not affected by the pragma of course...

**Comment by jneira** [comment]
I've just see 

> EDIT: the suppression was done through a coc-codeaction

@maralorn @googleson78 i think you are coc(n?vim) users, is it possible that some errors emitted by ghc and hls could be filtered out by the editor configuration? do you know how could be it reversed if that is the case?

**Reply by maralorn** [reply]
I don‘t know of any such coc feature. But I wouldn‘t claim to be an expert.

@Rintse: The only way to suppress warnings with a code action that I know about are code actions which let you add ghc options. So they would be in a comment somewhere in that file. That doesn‘t match that stack does not suppress the warnings but it‘s the best I can think of.

Other than that I would probably do something crazy like. `fd coc ~/.*` to find any caches …
